[2023-08-13 13:12:37,331] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: submit_pyspark_script_to_emr.execute_pyspark_script_sensor manual__2023-08-13T13:12:30.141639+00:00 [queued]>
[2023-08-13 13:12:37,354] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: submit_pyspark_script_to_emr.execute_pyspark_script_sensor manual__2023-08-13T13:12:30.141639+00:00 [queued]>
[2023-08-13 13:12:37,356] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-13 13:12:37,357] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-13 13:12:37,358] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-13 13:12:37,446] {taskinstance.py:1377} INFO - Executing <Task(EmrStepSensor): execute_pyspark_script_sensor> on 2023-08-13 13:12:30.141639+00:00
[2023-08-13 13:12:37,455] {standard_task_runner.py:52} INFO - Started process 224 to run task
[2023-08-13 13:12:37,460] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'submit_pyspark_script_to_emr', 'execute_pyspark_script_sensor', 'manual__2023-08-13T13:12:30.141639+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/orchestration/submit_spark_job_to_emr.py', '--cfg-path', '/tmp/tmpf8rtfe5y', '--error-file', '/tmp/tmpiix7je3q']
[2023-08-13 13:12:37,462] {standard_task_runner.py:80} INFO - Job 88: Subtask execute_pyspark_script_sensor
[2023-08-13 13:12:37,566] {task_command.py:370} INFO - Running <TaskInstance: submit_pyspark_script_to_emr.execute_pyspark_script_sensor manual__2023-08-13T13:12:30.141639+00:00 [running]> on host airflow_scheduler
[2023-08-13 13:12:37,723] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=submit_pyspark_script_to_emr
AIRFLOW_CTX_TASK_ID=execute_pyspark_script_sensor
AIRFLOW_CTX_EXECUTION_DATE=2023-08-13T13:12:30.141639+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-13T13:12:30.141639+00:00
[2023-08-13 13:12:37,736] {base.py:68} INFO - Using connection ID 'aws_default' for task execution.
[2023-08-13 13:12:37,741] {base_aws.py:210} INFO - Credentials retrieved from extra_config
[2023-08-13 13:12:37,742] {base_aws.py:100} INFO - Retrieving region_name from Connection.extra_config['region_name']
[2023-08-13 13:12:37,831] {emr.py:317} INFO - Poking step s-00791125JX60P5MM43I on cluster j-1Z3BJELOTWETV
[2023-08-13 13:12:38,164] {emr.py:74} INFO - Job flow currently PENDING
[2023-08-13 13:13:38,141] {emr.py:317} INFO - Poking step s-00791125JX60P5MM43I on cluster j-1Z3BJELOTWETV
[2023-08-13 13:13:38,527] {emr.py:74} INFO - Job flow currently FAILED
[2023-08-13 13:13:38,544] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/sensors/base.py", line 277, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/amazon/aws/sensors/emr.py", line 84, in poke
    raise AirflowException(final_message)
airflow.exceptions.AirflowException: EMR job failed for reason None with message Caused by: org.apache.hadoop.shaded.com.google.re2j.PatternSyntaxException: error parsing regexp: Unclosed group at pos 55: `{{ ti.xcom_pull(task_ids='fetch_cluster_id_and_paramns'` and log file s3://emr-data-947775527574/emr_logs/j-1Z3BJELOTWETV/steps/s-00791125JX60P5MM43I/stderr.gz
[2023-08-13 13:13:38,549] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=submit_pyspark_script_to_emr, task_id=execute_pyspark_script_sensor, execution_date=20230813T131230, start_date=20230813T131237, end_date=20230813T131338
[2023-08-13 13:13:38,564] {standard_task_runner.py:97} ERROR - Failed to execute job 88 for task execute_pyspark_script_sensor (EMR job failed for reason None with message Caused by: org.apache.hadoop.shaded.com.google.re2j.PatternSyntaxException: error parsing regexp: Unclosed group at pos 55: `{{ ti.xcom_pull(task_ids='fetch_cluster_id_and_paramns'` and log file s3://emr-data-947775527574/emr_logs/j-1Z3BJELOTWETV/steps/s-00791125JX60P5MM43I/stderr.gz; 224)
[2023-08-13 13:13:38,586] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-08-13 13:13:38,619] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
